app/services/symphony_client.rb

```diff
@@ -124,7 +124,7 @@ def get_hold_info(hold_key, session_token)
                           }
   end
 
-  def get_item_info(barcode, session_token, key)
+  def get_item_info(barcode = nil, session_token = nil, key = nil)
```

Comment:
@cdmo `session_token` should not be nil though, right?

Reply:
so i ran into a problem that i couldnt get around where if I only set key and barcode to nil ruby would throw

Reply:
```suggestion
  def get_item_info(session_token, barcode = nil, key = nil)
```

this seems to be working, needs to change in usages as well

Reply:
so the order matters then I guess? like it can't be `(barcode = nil, session_token, key = nil)` I know